Assets/Scripts/Localization/LocalizationTable.cs

```diff
@@ -59,7 +61,7 @@ public static void LoadLocalizationFile(string path)
          * </para>
          * </summary>
          */
-        public static void LoadLocalizationFile(string path, string localizationCode)
+        public static void LoadLocalizationFile(string path, string localizationName)
```

Comment:
I looked up the [table at msdn](https://msdn.microsoft.com/en-us/library/ee825488(v=cs.20).aspx) and it seems to use "name" for what we refer to as code. While code being just a number. I personally think we should stick to code. Are there any other documentation that names these values we can look at?